src/api/gitlab_payload_merge_request.txt

```diff
@@ -0,0 +1 @@
j@gmail.com"},"project":{"id":26212710,"name":"test","description":null,"web_url":"https://gitlab.com/eduardoj2/test","avatar_url":null,"git_ssh_url":"git@gitlab.com:eduardoj2/test.git","git_http_url":"https://gitlab.com/eduardoj2/test.git","namespace":"eduardoj","visibility_level":20,"path_with_namespace":"eduardoj2/test","default_branch":"master","ci_config_path":"","homepage":"https://gitlab.com/eduardoj2/test","url":"git@gitlab.com:eduardoj2/test.git","ssh_url":"git@gitlab.com:eduardoj2/test.git","http_url":"https://gitlab.com/eduardoj2/test.git"},"object_attributes":{"assignee_id":null,"author_id":8772703,"created_at":"2021-04-27 15:22:04 UTC","description":"","head_pipeline_id":null,"id":97617857,"iid":3,"last_edited_at":null,"last_edited_by_id":null,"merge_commit_sha":null,"merge_error":null,"merge_params":{"force_remove_source_branch":"1"},"merge_status":"preparing","merge_user_id":null,"merge_when_pipeline_succeeds":false,"milestone_id":null,"source_branch":"nuevo","source_project_id":26212710,"state_id":1,"target_branch":"master","target_project_id":26212710,"time_estimate":0,"title":"Nuevo","updated_at":"2021-04-27 15:22:04 UTC","updated_by_id":null,"url":"https://gitlab.com/eduardoj2/test/-/merge_requests/3","source":{"id":26212710,"name":"test","description":null,"web_url":"https://gitlab.com/eduardoj2/test","avatar_url":null,"git_ssh_url":"git@gitlab.com:eduardoj2/test.git","git_http_url":"https://gitlab.com/eduardoj2/test.git","namespace":"eduardoj","visibility_level":20,"path_with_namespace":"eduardoj2/test","default_branch":"master","ci_config_path":"","homepage":"https://gitlab.com/eduardoj2/test","url":"git@gitlab.com:eduardoj2/test.git","ssh_url":"git@gitlab.com:eduardoj2/test.git","http_url":"https://gitlab.com/eduardoj2/test.git"},"target":{"id":26212710,"name":"test","description":null,"web_url":"https://gitlab.com/eduardoj2/test","avatar_url":null,"git_ssh_url":"git@gitlab.com:eduardoj2/test.git","git_http_url":"https://gitlab.com/eduardoj2/test.git","namespace":"eduardoj","visibility_level":20,"path_with_namespace":"eduardoj2/test","default_branch":"master","ci_config_path":"","homepage":"https://gitlab.com/eduardoj2/test","url":"git@gitlab.com:eduardoj2/test.git","ssh_url":"git@gitlab.com:eduardoj2/test.git","http_url":"https://gitlab.com/eduardoj2/test.git"},"last_commit":{"id":"4b486afefa44177f23b4388d2147ae42407e7f64","message":"Nuevo\n","title":"Nuevo","timestamp":"2021-04-27T17:21:09+02:00","url":"https://gitlab.com/eduardoj2/test/-/commit/4b486afefa44177f23b4388d2147ae42407e7f64","author":{"name":"Eduardo Navarro","email":"enavarro@suse.com"}},"work_in_progress":false,"total_time_spent":0,"human_total_time_spent":null,"human_time_estimate":null,"assignee_ids":[],"state":"opened","action":"open"},"labels":[],"changes":{"merge_status":{"previous":"unchecked","current":"preparing"}},"repository":{"name":"test","url":"git@gitlab.com:eduardoj2/test.git","description":null,"homepage":"https://gitlab.com/eduardoj2/test"}}
```

Comment:
This file and a few others are unrelated to `Add missing CSS word break in build results`.

Reply:
:facepalm: fixed!